adapter/clickhouse/migrations/000035_set_withdraw_address_message.up.sql

```diff
@@ -0,0 +1,27 @@
+-- 000035_set_withdraw_address_message.up.sql
+
+CREATE TABLE IF NOT EXISTS spacebox.set_withdraw_address_message_topic
+(
+    height            Int64,
+    tx_hash           String,
+    msg_index         Int64,
+    delegator_address String,
+    withdraw_address  String
+) ENGINE = Kafka('kafka:9093', 'set_withdraw_address_message', 'spacebox', 'JSONEachRow');
+
+CREATE TABLE IF NOT EXISTS spacebox.set_withdraw_address_message
+(
+    height            Int64,
+    tx_hash           String,
+    msg_index         Int64,
+    delegator_address String,
+    withdraw_address  String
+) ENGINE = ReplacingMergeTree()
```

Comment:
Is it correct: `ReplacingMergeTree()`? Maybe need `ReplacingMergeTree(`tx_hash`, `msg_index`)` ?
Will it delete duplicates?